ipatests/test_xmlrpc/test_cert_plugin.py

```diff
@@ -224,7 +224,16 @@ def test_0009_cert_find(self):
         assert 'valid_not_before' in res
         assert 'valid_not_after' in res
 
-    def test_00010_cleanup(self):
+    def test_00010_san_in_cert(self):
+        """
+        Test if SAN extension is automatically added with default profile.
+        """
+        csr = self.generateCSR(str(self.subject))
+        res = api.Command[
+            'cert_request'](csr, principal=self.service_princ)['result']
+        assert 'san_dnsname' in res
+
+    def test_00011_cleanup(self):
```

Comment:
It would make sense to bump this to `test_99999_cleanup(self)` so that in the future when new
tests get added, this name won't also need to be changed (resulting in smaller, cleaner patches).